src/js/edu-benefits/components/SubmitMessage.jsx

```diff
@@ -63,11 +59,20 @@ export default class SubmitMessage extends React.Component {
               <span>{this.props.claimedBenefits}</span>
             </li>
             <li>
-              <b>Your relinquished benefits</b><br/>
+              <b>Benefits you gave up</b><br/>
               <span>{this.props.relinquishedBenefits}</span>
             </li>
           </ul>
         </div>
+        <h4>Documents</h4>
+        <p>This form doesn’t currently allow you to upload documents. Nothing is required right now. VA will review your application.</p>
+        <p><b>VA may eventually need the following documents:</b></p>
+        <ul>
+          <li></li>
```

Comment:
Ah I see what you're saying. So these items will be filled in conditionally based on the what the user selected? Until we build in the conditional logic, I might just list all possible things and then say (if applicable) next to ones that are more conditional, like the kicker document. Just so we don't have blanks. And yes, I would just use the placeholder language. Does that sound okay?